packages/usa-search/src/styles/_usa-search.scss

```diff
@@ -38,7 +38,8 @@ $search-icon: (
     }
 
     @media (forced-colors: active) {
-      border: 2px solid transparent;
+      border: 2px solid transparent; // Override forced-color button outline for to match input outline
```

Comment:
Can we create a shared variable for `2px solid transparent` that it can be used here as well?

Reply:
Ended up not needing to set a border at all after some refactoring!
Resolved in [249357d](https://github.com/uswds/uswds/pull/5147/commits/249357d55ecf814c75d1c7c843e96b4b48bbaaca)